### ISSUE TITLE
ascents missing from a facet search at some areas

### ISSUE THREAD

**Issue by scd** [theCrag/website#2381]
For example this area should have hundreds of ascents:

https://www.thecrag.com/climbing/australia/glasshouse-mountains/tibrogargan/area/12229003/ascents

But has none. There are plenty of ascents in one of the routes for the area:

https://www.thecrag.com/climbing/australia/glasshouse-mountains/tibrogargan/route/13447879/ascents

Note that the database seemed ok in dev???

**Comment by scd** [comment]
Glasshouse was moved, so my immediate thought was that the facets nodes lost their sync. I ran the fix script and there were not so many fixes made so this aspect is working ok. It did not fix the problem.

Secondly I tried fixing out of sync node collectors. Again not so many records needed correcting and it did not fix the problem.

Finally I reset the node cache for all everything under the sunshine coast and problem fixed. Obviously the move happened recently as the node cache fixes itself in about a week. Also there must be a dependency in the facets. The facets should not use the node cache, but probably does for gathering some input. I am guessing the depth of the original node.
